compiler/assembler.cpp

```diff
@@ -64,1326 +64,1322 @@ class CellWriter;
-  {133, "push.adr",   sIN_CSEG, parm1 },  /* version 4 */
-  { 37, "push.alt",   sIN_CSEG, parm0 },
-  { 39, "push.c",     sIN_CSEG, parm1 },
-  { 36, "push.pri",   sIN_CSEG, parm0 },
-  { 41, "push.s",     sIN_CSEG, parm1 },
-  {139, "push2",      sIN_CSEG, parm2 },  /* version 9 */
-  {141, "push2.adr",  sIN_CSEG, parm2 },  /* version 9 */
-  {138, "push2.c",    sIN_CSEG, parm2 },  /* version 9 */
-  {140, "push2.s",    sIN_CSEG, parm2 },  /* version 9 */
-  {143, "push3",      sIN_CSEG, parm3 },  /* version 9 */
-  {145, "push3.adr",  sIN_CSEG, parm3 },  /* version 9 */
-  {142, "push3.c",    sIN_CSEG, parm3 },  /* version 9 */
-  {144, "push3.s",    sIN_CSEG, parm3 },  /* version 9 */
-  {147, "push4",      sIN_CSEG, parm4 },  /* version 9 */
-  {149, "push4.adr",  sIN_CSEG, parm4 },  /* version 9 */
-  {146, "push4.c",    sIN_CSEG, parm4 },  /* version 9 */
-  {148, "push4.s",    sIN_CSEG, parm4 },  /* version 9 */
-  {151, "push5",      sIN_CSEG, parm5 },  /* version 9 */
-  {153, "push5.adr",  sIN_CSEG, parm5 },  /* version 9 */
-  {150, "push5.c",    sIN_CSEG, parm5 },  /* version 9 */
-  {152, "push5.s",    sIN_CSEG, parm5 },  /* version 9 */
-  { 48, "retn",       sIN_CSEG, parm0 },
-  { 74, "sdiv.alt",   sIN_CSEG, parm0 },
-  {104, "sgeq",       sIN_CSEG, parm0 },
-  {103, "sgrtr",      sIN_CSEG, parm0 },
-  { 65, "shl",        sIN_CSEG, parm0 },
-  { 69, "shl.c.alt",  sIN_CSEG, parm1 },
-  { 68, "shl.c.pri",  sIN_CSEG, parm1 },
-  { 66, "shr",        sIN_CSEG, parm0 },
-  { 71, "shr.c.alt",  sIN_CSEG, parm1 },
-  { 70, "shr.c.pri",  sIN_CSEG, parm1 },
-  {102, "sleq",       sIN_CSEG, parm0 },
-  {101, "sless",      sIN_CSEG, parm0 },
-  { 72, "smul",       sIN_CSEG, parm0 },
-  { 88, "smul.c",     sIN_CSEG, parm1 },
-  { 22, "sref.s.alt", sIN_CSEG, parm1 },
-  { 21, "sref.s.pri", sIN_CSEG, parm1 },
-  { 67, "sshr",       sIN_CSEG, parm0 },
-  { 44, "stack",      sIN_CSEG, parm1 },
-  { 16, "stor.alt",   sIN_CSEG, parm1 },
-  { 23, "stor.i",     sIN_CSEG, parm0 },
-  { 15, "stor.pri",   sIN_CSEG, parm1 },
-  { 18, "stor.s.alt", sIN_CSEG, parm1 },
-  { 17, "stor.s.pri", sIN_CSEG, parm1 },
-  {164, "stradjust.pri", sIN_CSEG, parm0 },
-  { 24, "strb.i",     sIN_CSEG, parm1 },
-  { 79, "sub",        sIN_CSEG, parm0 },
-  { 80, "sub.alt",    sIN_CSEG, parm0 },
-  {132, "swap.alt",   sIN_CSEG, parm0 },  /* version 4 */
-  {131, "swap.pri",   sIN_CSEG, parm0 },  /* version 4 */
-  {129, "switch",     sIN_CSEG, do_switch }, /* version 1 */
-  {135, "sysreq.n",   sIN_CSEG, parm2 },  /* version 9 (replaces SYSREQ.d from earlier version) */
-  {161, "tracker.pop.setheap", sIN_CSEG, parm0 },
-  {160, "tracker.push.c", sIN_CSEG, parm1 },
-  { 35, "xchg",       sIN_CSEG, parm0 },
-  { 83, "xor",        sIN_CSEG, parm0 },
-  { 91, "zero",       sIN_CSEG, parm1 },
-  { 90, "zero.alt",   sIN_CSEG, parm0 },
-  { 89, "zero.pri",   sIN_CSEG, parm0 },
-  { 92, "zero.s",     sIN_CSEG, parm1 },
+    /* node for "invalid instruction" */

```

Comment:
This is probably better before.

Reply:
Agree, I'll disable clang-format around this list.